src/api/tasks/tasksApi.ts

```diff
@@ -97,7 +94,7 @@ export const useAddTasks = (subSite?: string, pcolId?: string) => {
       if (batch.items.length > 0) {
```

Comment:
While this seemed like it would make sense to work, in reality it doesn't. Added a Final task when I had no Peer Reiveiewers.